apps/cms/src/admin/components/TotalCustomers.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+import { Avatar, Box, Card, CardContent, Grid, Typography } from '@mui/material';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import PeopleIcon from '@mui/icons-material/PeopleOutlined';
+
+const TotalCustomers = (props) => {
+    return (
+    <Card {...props}>
```

Comment:
Is it possible to change the background color of the card to suit the dark theme eg: by adding
`style={{
        backgroundColor: "black",
      }}
` 
You can also remove the `{...props}` as  it is implicitly passed down